infra/base-images/base-builder/detect_repo_test.py

```diff
@@ -23,11 +23,14 @@
 import unittest
 
 import detect_repo
+
 # Appending to path for access to repo_manager module.
+#pylint: disable=wrong-import-position
 sys.path.append(
-    os.path.dirname(
-        os.path.dirname(os.path.dirname(os.path.realpath(__file__)))))
+    os.path.dirname(os.path.dirname(os.path.dirname(
+        os.path.realpath(__file__)))))
```

Comment:
Do we want abspath or realpath here? I don't think we expect this file to be a symbolic link

Reply:
Done.